client/pages/queue/[section].vue

```diff
@@ -344,31 +357,29 @@ const filteredDocuments = computed(() => {
     case 'enqueuing':
       docs = documents.value.filter((d: any) => d.disposition === 'created')
       break
-    case 'pending':
-      docs = documents.value.filter(
-        (d: any) =>
-          d.disposition === 'in_progress' && d.assignmentSet?.length === 0
-      )
-      break
-    case 'exceptions':
-      docs = documents.value.filter(
-        (d: any) =>
-          d.disposition === 'in_progress' &&
-          d.labels?.filter((lbl: any) => lbl.isException).length
-      )
-      break
-    case 'inprocess':
+    case 'queue':
       docs = documents.value
         .filter(
           (d: any) =>
             d.disposition === 'in_progress' && d.assignmentSet?.length > 0
         )
+        .filter(
+          (d: any) => {
+            if(needsAssignment.value) {
+              return d.assignmentSet?.length === 0
+            }
+            if(hasException.value) {
+              return !!d.exception
+            }
+            return true
+        })
```

Comment:
I'm not sure this is quite what we need yet. With this implementation, if I read it correctly, the needsAssignment filter will hide the hasException filter when selected. The first return wins and the second condition is never evaluated.


The assigner should be able to look for documents needing assignment, seeing only those that have exceptions (to assign to people who can handle them), only those that don't have exceptions (for simpler assignments), or the full set of documents needing assignment whether the documents have exceptions or not.

Similarly, all of the users should be able to look at the documents that have exceptions, or don't have exceptions, whether the document is assigned or not.

That is, each of these filters should have [yes, no, either] semantics.

Reply:
Ok I've added a 'tristate' checkbox (see video on PR) and updated the filters.

Is it possible to move documents to this tab yet?

Reply:
documents should _definitely_ already be appearing in this tab.

Reply:
I think the issue was a bug you fixed in your last commit (bca2068) and the absurd slowness of the `queueList()` API call. The latter is fixed on main and on xfer. Do you now see documents on the queue?

Reply:
@jennifer-richards yes, now I am